src/main/scala/nl.knaw.dans.easy.springfield/Command.scala

```diff
@@ -42,6 +42,7 @@ object Command extends App
 
   type FeedBackMessage = String
 
+  private val prologue = "<?xml version='1.0' encoding='UTF-8'?>"
```

Comment:
It's called a ["prolog"](http://sbndev.astro.umd.edu/wiki/Anatomy_of_the_XML_Prolog)